src/GraphQL/ParseGraphQLSchema.js

```diff
@@ -25,18 +25,21 @@ const RESERVED_GRAPHQL_TYPE_NAMES = [
   'Query',
   'Mutation',
   'Subscription',
-  'ObjectsQuery',
-  'UsersQuery',
-  'ObjectsMutation',
-  'FilesMutation',
-  'UsersMutation',
-  'FunctionsMutation',
   'Viewer',
   'SignUpFieldsInput',
   'LogInFieldsInput',
 ];
-const RESERVED_GRAPHQL_OBJECT_QUERY_NAMES = ['get', 'find'];
-const RESERVED_GRAPHQL_OBJECT_MUTATION_NAMES = ['create', 'update', 'delete'];
+const RESERVED_GRAPHQL_QUERY_NAMES = ['health', 'viewer', 'get', 'find'];
+const RESERVED_GRAPHQL_MUTATION_NAMES = [
+  'signUp',
+  'logIn',
+  'logOut',
+  'createFile',
+  'call',
```

Comment:
I suggest to rename it to: `callCloudCode`

Reply:
done